src/renderer/components/wallet/settings/WalletSettings.tsx

```diff
@@ -169,37 +172,46 @@ export const WalletSettings: React.FC<Props> = (props): JSX.Element => {
     ]
   )
 
-  const [verifyAddressModalVisible, setVerifyAddressModalVisible] = useState(false)
-  const [addressToVerify, setAddressToVerify] = useState('')
-
+  const DEFAULT_ADDRESS_TO_VERIFY = useMemo(() => {
+    return {
+      address: '',
+      chain: THORChain
+    }
+  }, [])
+  const [addressToVerify, setAddressToVerify] = useState(DEFAULT_ADDRESS_TO_VERIFY)
```

Comment:
`useState<O.Option<Address>>` and you don't need an empty `DEFAULT_ADDRESS_TO_VERIFY` (which might be confusing)